packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -4,17 +4,52 @@
  */
 
 import child_process from "child_process";
+import fs from "fs";
 import commander from "commander";
 import { TestDriverTypes } from "@fluidframework/test-driver-definitions";
 import { ILoadTestConfig } from "./testConfigFile";
 import { createTestDriver, getProfile, initialize, safeExit } from "./utils";
 
+interface ITestUserConfig {
+    /* Credentials' key/value description:
+     * Key    : Username for the client
+     * Value  : Password specific to that username
+     */
+    credentials: Record<string, string>;
+}
+
+async function getTestUsers(credFile?: string) {
+    if (credFile === undefined || credFile.length === 0) {
+        return undefined;
+    }
+
+    let config: ITestUserConfig;
+    try {
+        config = JSON.parse(await new Promise<string>((resolve, reject) =>
+            fs.readFile("./testUserConfig.json", "utf8", (err, data) => {
```

Comment:
why not just use readFileSync

Reply:
@anthony-murphy Its just as good practice for application using async await, I prefer async version of read file. Since its not a UI application even readFileSync also works here, changed.
Also, fixed the bug where file name was still hard coded.